test/smoke/src/sql/main.ts

```diff
@@ -76,6 +77,15 @@ export async function setup(app: ApplicationOptions): Promise<void> {
 
 const root = path.dirname(path.dirname(path.dirname(path.dirname(__dirname))));
 
+// Gets the appropriate ADS Sqlite release version for web and non web smoke tests. Web smoke tests are
```

Comment:
A couple suggestions here : 

1. For typescript/javascript you should always be using the JS-doc format (type `/**` and then hit enter to auto-populate the template)
2. It's best to avoid putting stuff like specific version numbers that can change in comments like this - since that just means that if someone updates the version they have to now update it in multiple places (and often comments are missed). It's fine just to have the description here generically say that it is getting the appropriate
3. The function is called `getADSSqliteVersionForWeb` but then takes in a boolean which tells it whether it's web or not, in this case I think `getSqliteDownloadUrl` is a much better name since that's more clear about what it's doing

Reply:
Thanks for the feedback, I will definitely keep this in mind going forward.